app/views/initiatives/_initiatives.html.erb

```diff
@@ -1,23 +1,13 @@
 <div class="init-list">
   <% initiatives.each do |initiative| %>
-    <%= link_to edit_initiative_path(initiative), :style=> "text-decoration: none;" do %>
-      <div class="card">
-        <h3 class="init-name"><%= initiative.title %></h3>
-        <p class="init-description"><%= initiative.description %></p>
-        <div>
-          <div class="init-footer w-row">
-            <div class="w-col w-col-2 padding-zero"><%= image_tag "icon-#{initiative.completion ? 'completion': 'calendar'}.svg", class: "icon-target" %></div>
-            <div class="card-date w-col w-col-10">
-              <%= initiative.target_date.to_s(:long_ordinal) unless initiative.target_date.nil? %>
-            </div>
-            <div class="w-col w-col-2 padding-zero">
-              <%= link_to initiative_path(initiative), method: :delete, data: { confirm: "Are you sure you would like to delete Initative: #{initiative.title}" }, :class => "no-link", :id => "delete-initiative" do %>
-                <%= image_tag "icon-delete.svg" %>
-              <% end %>
-            </div>
-          </div>
-        </div>
-      </div>
+    <% if current_user.team_members.pluck(:id).include?(params[:user_id].to_i) %>
```

Comment:
How can we move this logic out of the view?

Reply:
change the logic